### ISSUE TITLE
Update to Java 11

### ISSUE THREAD

**Issue by mdbell** [mdbell/Noexes#24]
Java 10 is old, need to update.

**Comment by EphraimBane** [comment]
Exactly. Also Oracle won't let me get Java 10 anymore without paying them.

**Comment by Slluxx** [comment]
This is a big issue.

**Comment by luiscesjr** [comment]
> Exactly. Also Oracle won't let me get Java 10 anymore without paying them.

Here it's free:

https://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase10-4425482.html

Just need to create a free account if you don't have already and choose the Java SE Runtime Environment 10.0.2, which is the last JRE provided by Oracle. Using OpenJDK or  Amazon Corretto to create a JRE is a lot more work, so I suggest using the 10.0.2 for now from that link

**Comment by dually8** [comment]
I've been able to get this running on Java 11, but packaging it is proving to be super complicated. Apparently the deploy function was taken out of JavaFX 11, so there's this super convoluted way you're supposed to package stuff now. It's a big ol' oof.

**Comment by Stuntaneous** [comment]
It should be noted in the readme that this very specifically needs JRE v10. Like, explicitly say newer versions won't work. 

Just for anyone googling, I've got it going at last with 10.0.2 (x64) on Win10 x64

**Comment by mdbell** [comment]
I just pushed a commit to make it so the project itself compiles with JDK 16, eventually we'll have a full build process to link the modules, as well as using jpackage to create platform specific executables

**Comment by theboy181** [comment]
when?

**Comment by RedStoneMatt** [comment]
Considering the "Building the client" sections is empty, it'd be nice to have a build, because I can't figure out how to make one.
